### ISSUE TITLE
Can't use styled-jsx/css in custom webpack loader because "Module not found"

### ISSUE THREAD

**Issue by tom2strobl** [zeit/styled-jsx#324]
The new v2 release looked promising to finally get our desired behaviour back: having normal 
 `.scss` files, importing them as a `import styles from "./component.scss"` object and putting this object into the `<style jsx>{styles}</style>`.

The first step works:

```javascript
import css from 'styled-jsx/css'
export default css`
  .wrapper {
    color: darken(#fff, 50%);
  }
`
```

Importing this `style.js` file as mentioned above works! But it needs the .js extension and import css / export css magic. A custom loader should do the trick! So we modified our next.js config to use a loader for .scss files:

```javascript
// .babelrc
{"presets": [["next/babel",{"styled-jsx": {"plugins": [
  "styled-jsx-plugin-sass"
]}}]]}
// next.config.js
exports.webpack = (config, { buildId, dev }) => {
  config.module.rules.push({
    test: /\.scss/,
    use: 'styled-jsx-scss-loader'
  });
  // push in our custom loader that lives in an internals folder
  config.resolveLoader.modules.push('internals');
  return config;
}
// internals/styled-jsx-scss-loader.js
module.exports = function(source) {
  this.cacheable && this.cacheable();
  this.value = source;
  return `import css from 'styled-jsx/css'; export default css\`${JSON.stringify(source)}\``;
}
```

But it doesn't work since it throws a `Module not found: Error: Can't resolve 'styled-jsx/css' `. I then read somewhere between issue comments that this is a fake import (boo).

Long story short: Do you think there is a way to accomplish what we seek for, or will it never work because of that "fake import" business?

**Comment by giuseppeg** [comment]
Similar to https://github.com/zeit/styled-jsx/issues/322 Closing as duplicate
